### ISSUE TITLE
Table support in text cells?

### ISSUE THREAD

**Issue by olibclarke** [HappenApps/Quiver#1151]
Hi @ylian,

Would it be possible to add support for simple tables in text cells? I know it has been raised before, but it is a simple feature that would make recording data much easier in Quiver. If I'm just recording data, I don't want to have to think about formatting in Markdown...

The implementation in Apple's Notes.app is simple and works very well - basically consists of one insert table button, with addition/deletion of rows/columns handled by right-click context menus - would something like this be possible? It would only mean including an insert table button as a single addition to the formatting buttons, so I don't think it would add much clutter to the UI, and I think the functionality would be very useful for many users (particularly users like myself who are scientists using Quiver as a lab notebook).

Alternatively, adding a separate cell type (Spreadsheet cell? Table cell?) might be another way to go? What do you think?

Cheers
Oli

**Comment by ylian** [comment]
Yes, the Apple Note's design is quite good. Simple but powerful.

I am focusing on pushing out Quiver 3.2 (model layer rewrite) at the moment. Will get to other features as soon as 3.2 is out.

**Comment by jlgerber** [comment]
Love the app but this is an obvious gap. Hopefully, this can be added soon.

**Comment by daharper** [comment]
Yeah, I have to agree.

Great app, but the ability to add a table is critical.

Any ETA on when, if at all, this feature will be added?

Thank you.

**Comment by philwantsfish** [comment]
+1, please give us table support

**Comment by nullskill** [comment]
5 years later... no tables, nothing. The app is abandoned. That's sad :(
